File: freecodecamp/daily-code-challenge/2026-02-10.js
function getRelativeResults(results) {
  // helper: convert H:MM:SS to seconds
  const toSeconds = (time) => {
    const [h, m, s] = time.split(":").map(Number);
    return h * 3600 + m * 60 + s;
  };

  const winnerTime = toSeconds(results[0]);

  return results.map((time, index) => {
    if (index === 0) return "0";

    const diff = toSeconds(time) - winnerTime;
    const minutes = Math.floor(diff / 60);
    const seconds = diff % 60;

    return `+${minutes}:${String(seconds).padStart(2, "0")}`;
  });
}

File: freecodecamp/daily-code-challenge/2026-02-10.py
def get_relative_results(results):
    # helper: convert H:MM:SS to seconds
    def to_seconds(time):
        h, m, s = map(int, time.split(":"))
        return h * 3600 + m * 60 + s

    winner_time = to_seconds(results[0])
    output = []

    for i, time in enumerate(results):
        if i == 0:
            output.append("0")
        else:
            diff = to_seconds(time) - winner_time
            minutes = diff // 60
            seconds = diff % 60
            output.append(f"+{minutes}:{seconds:02d}")

    return output
